l10n_it_fatturapa_out/wizard/efattura.py

```diff
@@ -206,7 +206,14 @@ def fpa_to_eur(amount, invoice):
             # check value code
             code = self.partner_id.ipa_code
         else:
-            code = self.partner_id.codice_destinatario
+            if (
```

Comment:
piuttosto che modificare `l10n_it_fatturapa_out` con questa logica, io fare una compute nella definizione del campo `codice_destinatario` e implementerei la logica lì, dove è definito il campo (`l10n_it_fatturapa`).

infatti in futuro un altro modulo potrebbe usare il `codice_destinatario` - se va popolato da un partner padre deve reimplementare la logica